Search-Engine-Bandits/nutshell
# some final touches

# Description

added nav bar with logout button
added some css with background image and colors

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout sh-messing-with-styling
npm start in main directory if not already started
navbar with image and logout button should appear
ability to click logout button will take you back to welcome page
once logged in, dashboard view will show with all your current tasks, news, etc

# Checklist:
